### PR TITLE
Update to catch false positive file locations in database for downlads

### PR DESCRIPTION
**Pull request by isuftin**
Followup from doing this for HEAD requests

### CODE REVIEW

File: coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/download/DownloadUtility.java [hunk 1]
```diff
@@ -116,7 +116,7 @@ public static boolean stageItemDownload(Item stageThis, File stagingDir) throws
 		try {
 			Map<WFSService, SingleDownload> downloadMap = new HashMap<>();
 			populateDownloadMap(downloadMap, stageThis);
-			List<String> namesUsed = new ArrayList<>();
+			List<String> namesUsed = new ArrayList<>(downloadMap.values().size());
```
Comment (jiwalker-usgs) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/download/DownloadUtility.java]: Over-optimize much?

File: coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/jpa/DownloadManager.java [hunk 2]
```diff
@@ -122,10 +123,8 @@ public void deleteAllMissing() {
 	}
 
 	public List<Download> getAllStagedDownloads() {
-		List<Download> resultList;
 		Query selectQuery = em.createQuery(HQL_SELECT_ALL);
-		resultList = selectQuery.getResultList();
-		return resultList;
+		return selectQuery.getResultList();
```
Comment (jiwalker-usgs) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/jpa/DownloadManager.java]: fair enough

File: coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/model/util/Download.java [hunk 3]
```diff
@@ -5,6 +5,8 @@
 import java.net.URI;
 import java.net.URISyntaxException;
 import java.util.Date;
+import java.util.logging.Level;
```
Comment (jiwalker-usgs) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/model/util/Download.java]: JULI?

**Comment by jiwalker-usgs** [comment]
Otherwise looks good. :shipit:

**Comment by jread-usgs** [comment]
what's a `downlad`?

File: coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/model/util/Download.java (hunk 3 above)
Comment (isuftin) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/model/util/Download.java]: Ahh... Hmm.

File: coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/download/DownloadUtility.java (hunk 1 above)
Comment (isuftin) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/download/DownloadUtility.java]: http://www.brainphysics.com/images/ocdcycle2.gif
image

**Comment by isuftin** [comment]
@jread-usgs Ever watch Life Goes On?

**Comment by jread-usgs** [comment]
`lads`

**Pull request merged**